.travis.yml

```diff
@@ -4,6 +4,9 @@ dist: trusty
 node_js:
   - 6
```

Comment:
I think we should only use v12 and v14. LTS for node v10 was until april 2020. https://nodejs.org/ru/blog/release/v10.13.0/

Reply:
But I'm still sure that we need to support older versions to prevent accidental regressions.

Reply:
Yep, but since we have a breaking change with the change that we drop "Some double quotes aren't necessary", we have to increase the version number anyways. So this change would only affect version 2.0.0. We can not guarantee support of a node version that's not supported anymore by them self. So if someone needs support for node version 6, the person can use version 1 of this package. For most of the cases version 2 is still usable for older versions but we do not support them explicitly anymore.